templates/composer.yml

```diff
@@ -109,6 +111,32 @@ objects:
             mountPath: "/var/lib/osbuild-composer"
           - name: cache-directory
             mountPath: "/var/cache/osbuild-composer"
+        - image: "fluentd-hec:1.2.13"
+          name: fluentd-sidecar
+          resources:
+                requests:
+                  cpu: "${CPU_REQUEST}"
+                  memory: "${MEMORY_REQUEST}"
+                limits:
+                  cpu: "${CPU_LIMIT}"
+                  memory: "${MEMORY_LIMIT}"
```

Comment:
Probably a theoretical concern, but can we limit the cpu to 200m (same as the request)? Composer's limit it 1, and the namespace limit is 3 in total. I just never want to run into a situation where fluentd would slow down composer by taking up the cpu quota.